CompuRacer_Core/main.py

```diff
@@ -57,22 +56,37 @@
 __email__ = "rvanemous@computest.nl"
 __status__ = "Prototype"
 
-# -------------- main client program & server -------------- #
-if __name__ == '__main__':
+# ---  Maintenance Developer information --- #
```

Comment:
Ik zou deze twee author/developer info stukjes samenvoegen. Volgens mij is het niet de bedoeling dat je dubbele email/version/license velden hebt namelijk. Check bijv. ff hier: https://stackoverflow.com/questions/1523427/what-is-the-common-header-format-of-python-files